components/html-block/html-block.js

```diff
@@ -203,6 +203,8 @@ class HtmlBlock extends RtlMixin(LitElement) {
 		super.firstUpdated(changedProperties);
 
 		if (this._renderContainer) return;
+
+		// The d2l-html-block-rendered class is used to apply CSS outside of the html-block component. Do not change lightly.
```

Comment:
There's already a test that will implicitly fail if we change this class. It's relying on querying `.d2l-html-block-rendered` and returning something with valid inner HTML.